preppal-be/routes/userApi.ts

```diff
@@ -17,32 +19,53 @@ router.get("/", async (req, res) => {
 });
 
 
-router.post("/createUser", async (req, res) => {
-    try {
-        const errors = validationResult(req);
+/**
+ * GET - Get an account with corresponding username
+ */
+routerUserApi.get("/:username", async (req, res) => {
+    let username = req.params.username;
+    let password = req.params.password;
+    const user = await User.find( { username: username}).select("-password");
```

Comment:
why do we need the password here if we're just grabbing the account username and checking if they exist or not?

Reply:
a relic, I was thinking about doing login auth here originally and forgot to remove it